internal/encoding/gotypes/generate.go

```diff
@@ -217,6 +231,9 @@ type generator struct {
 
 	// def tracks the generation state for a single CUE definition.
 	def *generatedDef
+
+	// importAliases maps package names to the alias
+	importAliases map[string]string
```

Comment:
Note that we organize fields on a per-package and per-definition basis. This should be on a per-package basis I assume.